app/src/main/java/com/hewgill/android/nzsldict/SignVideoFragment.java

```diff
@@ -83,9 +127,13 @@ public void onPrepared(MediaPlayer mp) {
             }
         });
 
+        // Start loading video if network is available
+        if (checkConnectivity()) mVideo.setVideoURI(Uri.parse(mDictItem.video));
```

Comment:
Maybe the `mVideo.setVideoURI(Uri.parse(mDictItem.video))` can move into the method. name it `attemptToPlayVideo()`